### ISSUE TITLE
"View on Instagram" button is cropped out

### ISSUE THREAD

**Issue by sharunkumar** [igdmapps/igdm#1212]
**Describe the bug**
When opening a post in a chat, the "View on Instagram" button is out of scope of the window

**To Reproduce**
1. Log In
2. Open a chat where someone sent you a post
3. Open the post

**Expected behavior**
View on Instagram button is fully visible

**Screenshots**
![image](https://user-images.githubusercontent.com/715417/74550058-b9ca4e80-4f76-11ea-8795-ca373b7e2d40.png)

**Desktop (please complete the following information):**
 - OS: Windows
 - Version: Windows 10

**App version (please complete the following information):**
 - Version: 2.8.1

**Comment by sharunkumar** [comment]
So I just realized this goes down because of the caption, but it would be better if this was a floating button 😅

**Comment by ifedapoolarewaju** [comment]
but if you scroll down, you are still able to access the button right?

**Comment by sharunkumar** [comment]
Yeah its just that you have to do it for every post. Probably should have marked this as a feature request to make the button floating
